data/content_scripts/visit.js

```diff
@@ -1,6 +1,3 @@
-var SERVER_URL = "https://chronicle.dev.mozaws.net";
-var CREATE_VISIT_ENDPOINT = SERVER_URL + "/v1/visits";
```

Comment:
it might be nice to keep the create endpoint in the config file, since, someday, we will move to v2 of the API. But that's a far distant galaxy

edit: I hate when I accidentally comment on the commit instead of the PR :-P